src/test/java/com/example/project/test/EmployeeTest.java

```diff
@@ -0,0 +1,137 @@
+package com.example.project.test;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.example.project.dto.*;
+import io.restassured.RestAssured;
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.builder.ResponseSpecBuilder;
+import io.restassured.filter.log.RequestLoggingFilter;
+import io.restassured.filter.log.ResponseLoggingFilter;
+import io.restassured.specification.RequestSpecification;
+import io.restassured.specification.ResponseSpecification;
+import lombok.val;
+import org.apache.commons.lang3.StringUtils;
+import org.assertj.core.api.SoftAssertions;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import static io.restassured.RestAssured.given;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class EmployeeTest {
+
+    private RequestSpecification requestSpecification;
+    private ResponseSpecification responseSpecification;
+    private final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
please use constructor for initialisation

Reply:
I'm not sure what you mean. Which one do you want to be initialized with a constructor? Specifications use builders and ObjectMapper is already initialized with a constructor.

Reply:
I meant to do something like below:

```
public class EmployeeTest {

    private RequestSpecification requestSpecification;
    private ResponseSpecification responseSpecification;
    private final ObjectMapper objectMapper;
    
   public EmployeeTest() {
       objectMapper = new ObjectMapper();
   }

```